mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -315,6 +317,12 @@
         The first parameter is the name of the app defined in app_name (for example: Fenix). -->
     <string name="browser_menu_delete_browsing_data_on_quit">Quit %1$s</string>
 
+    <!-- Menu "contextual feature recommendation" (CFR) -->
+    <!-- Text for the title in the contextual feature recommendation popup promoting the menu feature. -->
+    <string name="menu_cfr_title">New: streamlined menu</string>
+    <!-- Text for the message in the contextual feature recommendation popup promoting the menu feature. -->
+    <string name="menu_cfr_body">Find what you need faster, from private browsing to save actions.</string>
```

Comment:
I believe this is intended according to the content design in https://www.figma.com/design/RFz9fYtotQCQuinwcZujZt/Menu-Redesign?node-id=16563-182996&m=dev

Reply:
Pinged Content Design to be extra sure, thanks

Reply:
From EmilyW on Slack: "It's correct because we're referring to actions you use to save, not actions you yourself have saved, or anything about saved in the past tense."